server/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+let app = express();
+const db = require('../database/seeder.js');
```

Comment:
Are you re-seeding your db with new data each time your server/index.js runs this file? I need to look into whether this is a best practice or not. I don't think I am dropping the data re-seeding unless I manually invoke the seeding script in mine.

Reply:
Thank you. I will try to separate it into two files so the seeding won't happen every time.